src/pages/EditProfile/EditProfile.jsx

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react';
+import {
+    Box,
+    Container,
+    HStack,
+    VStack,
+    Avatar,
+    Image,
+    Text,
+    Grid,
+    Center,
+    Input,
+    Button,
+} from '@chakra-ui/react';
+// import proto from '../assets/proto.png';
+
+function EditProfile() {
+    const [fullName, setFullName] = useState('');
```

Comment:
I think we need separate fields/states for firstName and lastName instead of one fullName. You need also states to handle the profile and id images.